snapshot/snapshotfs/upload_test.go

```diff
@@ -124,7 +124,7 @@ func TestUpload(t *testing.T) {
 
 	policyTree := policy.BuildTree(nil, policy.DefaultPolicy)
 
-	s1, err := u.Upload(ctx, th.sourceDir, policyTree, snapshot.SourceInfo{})
+	s1, err := u.Upload(ctx, th.sourceDir, policyTree, snapshot.SourceInfo{}, nil)
```

Comment:
here and below: this change needs to be reverted, no?
```suggestion
	s1, err := u.Upload(ctx, th.sourceDir, policyTree, snapshot.SourceInfo{})
```